po/bn.po

```diff
@@ -0,0 +1,608 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the flatseal package.
+# ARITRA SAHA <CAPSAICIN@TUTA.IO>, 2024.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: flatseal\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-09-18 16:26-0400\n"
+"PO-Revision-Date: 2024-10-10 HO:MI+ZONE\n"
+"Last-Translator: ARITRA SAHA <capsaicin@tuta.io>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.desktop.in:4
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:4
+msgid "Flatseal"
+msgstr "Flatseal"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:10
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:10
+msgid "Manage Flatpak permissions"
+msgstr "ফ্ল্যাটপ্যাক অনুমতিসমূহ পরিচালনা করুন"
+
+#: data/com.github.tchx84.Flatseal.desktop.in:15
+msgid "seal;sandbox;override;"
+msgstr "seal;sandbox;override;"
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:12
+msgid """Flatseal is a graphical utility to review and modify permissions from your ""Flatpak applications."
+msgstr "ফ্ল্যাটসিল একটি গ্রাফিক্যাল ইউটিলিটি যা আপনার ফ্ল্যাটপ্যাক অ্যাপ্লিকেশনগুলির অনুমতি পর্যালোচনা এবং পরিবর্তন করতে ব্যবহৃত হয়।"
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:21
+msgid "The Flatseal main window"
+msgstr "Flatseal প্রধান উইন্ডো"
+
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:25
+msgid "Flatseal showing filesystem permissions"
+msgstr "ফ্ল্যাটসিল ফাইল সিস্টেম অনুমতি দেখাচ্ছে"
```

Comment:
Whether you keep it in English or Bengali, just be consistent through all the document.